easybuild/easyconfigs/e/Emacs/Emacs-28.2-GCCcore-12.2.0.eb

```
@@ -0,0 +1,50 @@
+#
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# reciPY derived from preexisting reciPy for GCCcore-9.2.0.
+# Modder:    Ben Langenberg <ben.langenberg@ufz.de>
+# Updated by: Thomas Eylenbosch (Gluo N.V.)
+
+easyblock = 'ConfigureMake'
+
+name = 'Emacs'
+version = '28.2'
+
+homepage = 'https://www.gnu.org/software/emacs/'
+description = """GNU Emacs is an extensible, customizable text editor--and more.
+ At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming
+ language with extensions to support text editing."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a6912b14ef4abb1edab7f88191bfd61c3edd7085e084de960a4f86485cb7cad8']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('pkg-config', '0.29.2'),
```

```suggestion
    ('pkgconf', '1.9.3'),
```

We are trying to move away from pkg-config to the (actually maintained) pkgconf instead.